wolfcrypt/src/integer.c

```diff
@@ -4729,12 +4729,9 @@ LBL_U:mp_clear (&v);
 #endif /* WOLFSSL_KEY_GEN */
 
 
-#if defined(HAVE_ECC) || defined(WOLFSSL_KEY_GEN) || defined(HAVE_COMP_KEY)
```

Comment:
Some compilers complain about unused constant static variables.
I would change the list of conditionals to match those around both uses.